build.gradle

```diff
@@ -0,0 +1,77 @@
+buildscript {
```

Comment:
shall we add license header to this file?

Reply:
Added